src/main/content/_assets/js/common-multipane.js

```diff
@@ -142,9 +142,7 @@ function createEndOfGuideContent(){
     // Concatenate the guide title and guide attribution license and append it to the end of guide.
     var guideAttribution = $("#guide-attribution").siblings().find('p').text();
```

Comment:
Code might be easier to understand if this var was named `guideAttributionText` to avoid confusion with the element.

Reply:
Fixed